features/adres.feature

```diff
@@ -20,22 +24,22 @@ Functionaliteit: Adresvelden vullen
       | burgerservicenummer | <burgerservicenummer>                        |
       | fields              | verblijfplaats                               |
       Dan heeft de response een persoon met alleen de volgende 'verblijfplaats' gegevens
-      | naam                      | waarde      |
-      | type                      | Adres       |
-      | functieAdres.code         | W           |
-      | functieAdres.omschrijving | woonadres   |
-      | verblijfadres.straat      | <straat>    |
-      | verblijfadres.korteNaam   | <korteNaam> |
+      | naam                              | waarde               |
+      | type                              | Adres                |
+      | functieAdres.code                 | W                    |
+      | functieAdres.omschrijving         | woonadres            |
+      | verblijfadres.officieleStraatnaam | <officielestraat>    |
+      | verblijfadres.korteStraatNaam     | <korteNaam>          |
 
       Voorbeelden:
-      | burgerservicenummer | straatnaam               | naam openbare ruimte                    | straat                                  | korteNaam                |
+      | burgerservicenummer | straatnaam               | naam openbare ruimte                    | officielestraat                         | korteNaam                |
       | 000000152           | Kappeyne v d Cappellostr | Annelien Kappeyne van de Coppellostraat | Annelien Kappeyne van de Coppellostraat | Kappeyne v d Cappellostr |
-      | 000000164           | Zomerdijkstraat          |                                         | Zomerdijkstraat                         | Zomerdijkstraat          |
+      | 000000164           | Zomerdijkstraat          |                                         |                                         | Zomerdijkstraat          |
 
   Rule: Voor een binnenlands adres wordt adresregel1 samengesteld conform NEN 5825:2002
-    - Veld adresregel1 wordt samengesteld uit korteNaam + aanduidingBijHuisnummer + huisnummer + huisletter + huisnummertoevoeging
+    - Veld adresregel1 wordt samengesteld uit korteStraatnaam + aanduidingBijHuisnummer + huisnummer + huisletter + huisnummertoevoeging
```

Comment:
ik denk dat je hier straatnaam moet gebruiken, niet korteStraatnaam. De input voor adresregel is immers wat wij in de scenario's in de Gegeven stap zetten.

Reply:
Aangepast